<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
               ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 748, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 247, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/check_syntax.py", line 42, in check_recipe
    self.message(section=f"{path}/{n}", data=True, output=output)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 329, in message
    message = self.make_message(
              ^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 378, in make_message
    sl, sc, el, ec = recipe.get_raw_range(section)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 497, in get_raw_range
    nodes, keys = self._walk(path)
                  ^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/recipe.py", line 478, in _walk
    nodes.append(last[key])
                 ~~~~^^^^^
TypeError: string indices must be integers, not 'str'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_59mzutaqte/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -26,7 +26,7 @@ test:
 # CUDA Profiling Tools Interface (CUPTI) Library
 about:
   home: https://developer.nvidia.com/cuda-toolkit
-  license: proprietary - Nvidia
+  license: LicenseRef-ProprietaryNvidia
   license_family: Proprietary
   summary: development environment for GPU-accelerated applications, CUPTI components
   description: |

